app/models/message.rb

```diff
@@ -20,4 +20,17 @@ class Message < ApplicationRecord
   belongs_to :conversation
 
   validates :text, presence: true
+
+  def notify
+    return if target_match
+
+    users = conversation.users.reject { |each| each == user }
+    NotifyRequestJob.perform_later(user, users, text)
+  end
+
+  private
+
+  def target_match
+    text == I18n.t('api.messages.target_match')
```

Comment:
So, you're creating a message when two users have a match, right?

Reply:
yes @gonzaM , with no user assigned